blazor/gantt-chart/rows.md

```diff
@@ -548,6 +548,115 @@ The following code example shows how to drag and drop a row on button click acti
 }
 ```
 
+### Dynamically hide rowdragdrop icons
```

Comment:
rowdragdrop is not a word.

Reply:
Also move Drag and drop topic itself to left side toc file. And update the URL used in sample browsers any other places also.